### DISCUSSION TITLE
What is the key value for a relation?

### DISCUSSION THREAD

**Discussion by rjordanriley** [vzhd1701/csv2notion#29] [General]
Where do I find that?  I am wanting to import  relation column.

**Comment by vzhd1701** [comment]
Do you have relation configured in the Notion table?

**Reply by rjordanriley** [reply]
Yes.  I suppose I am confused what the key is, of the pages in the target database.  I assume the title does not work.

**Reply by vzhd1701** [reply]
It is the title. Do you get "ERROR: CSV [row number]: Value 'column value' for relation 'column name [column] -> table name [DB]' is not a valid value."?

**Reply by rjordanriley** [reply]
Yes, exactly, that's the error.  Phew, it is the title?  That will be very helpful...

**Reply by rjordanriley** [reply]
What do I have wrong then?  Formatting?  Thank you...

**Reply by vzhd1701** [reply]
Yes, bad formatting is most probable cause, text should match exactly. Check for trailing spaces in the destination table title column values and your CSV file. Columns in CSV must be separated by comma without extra spaces.

Wrong:
```
Column 1, Column 2, Column 3
value11, value12, value13
value21, value22, value23
```

Correct:
```
Column 1,Column 2,Column 3
value11,value12,value13
value21,value22,value23
```

**Reply by rjordanriley** [reply]
I have double checked that and do not see any issue.  This is the CSV:

Title,Start,End,Case
value,value,value,Kateryna
value,value,value,Fanning
value,value,value,Serrano


And this is the error:

ERROR: CSV [2]: Value 'Kateryna' for relation 'Case [column] -> Matters [DB]' is not a valid value.
ERROR: CSV [3]: Value 'Fanning' for relation 'Case [column] -> Matters [DB]' is not a valid value.
ERROR: CSV [4]: Value 'Serrano' for relation 'Case [column] -> Matters [DB]' is not a valid value.

That's it, just one word page names in the matters database, and one word in the column to match.("Case").  The help is much appreciated!
